src/formats/yarle/utils/templates/default-template.ts

```diff
@@ -1,5 +1,7 @@
-const tagBlock = '{tags-array-block}---\ntags: {tags-array}\n---\n{end-tags-array-block}';
+const frontmatterDelimiter = '---\n';
+const sourceBlock = '{source-url-block}\nsource: {source-url}\n\n{end-source-url-block}';
```

Comment:
Will this cause the `source` property to be present even for files that do not have a source specified?

Reply:
Nope, it is removed from the template automatically if no source property specified in the note data structure